packages/block-library/src/footnotes/block.json

```diff
@@ -0,0 +1,18 @@
+{
+	"$schema": "https://schemas.wp.org/trunk/block.json",
+	"apiVersion": 3,
+	"name": "core/footnotes",
+	"title": "Footnotes",
```

Comment:
Should it be "Post Footnotes" like "Post Title", "Post Content", "Post Author"...

Reply:
What if we add support for pages? Or other custom post types?

Reply:
Not a strong opinion here, I also see that some of the "post *" blocks have been renamed.